.travis.yml

```diff
@@ -1,21 +1,23 @@
 language: python
 
+sudo: false
+
 python:
   - "2.7"
   - "2.6"
+  - "3.4"
+
+addons:
+  apt:
+    packages:
+      - libgeos-c1
+      - libxml2-dev
 
 install:
-  - sudo apt-get install -y libgeos-c1
   - pip install . --use-mirrors
-  - pip install -r ./requirements-standalone.txt
-  - pip install -r ./requirements-dev.txt
-  - python ./setup.py -q install
-
-before_script:
-  - sudo add-apt-repository ppa:gcpp-kalxas/ppa-tzotsos -y
```

Comment:
That explains the change in behaviour. I am also guessing that libxml 2.8 included in this new environment has parts of 2.9 backported...